pages/linux/ss.md

```diff
@@ -30,3 +30,7 @@
 - Show all TCP IPv4 sockets locally connected on the subnet 192.168.0.0/16:
 
 `ss -4t src {{192.168/16}}`
+
+- Kill IPv4 / IPv6 Socket Connection with destination IP 192.168.1.17 and destination port 8080:
+
+`ss -K dst {{192.168.1.17}} dport = {{8080}}`
```

Comment:
```suggestion
- Kill IPv4 or IPv6 Socket Connection with destination IP 192.168.1.17 and destination port 8080:

`ss --kill dst {{192.168.1.17}} dport = {{8080}}`
```

Optional fix: `or` may sound better.
Mandatory fix: [long option must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).

Reply:
All other examples use the short option, I don't why only this one should use the long option, I think the description of the command it's pretty clear

Reply:
> All other examples use the short option, I don't why only this one should use the long option, I think the description of the command is pretty clear as well

All other examples are incorrect where they don't use portable long options and must be fixed. If you wanna change rules - u can suggest any edits. :)

Reply:
I followed the same convention of the other examples, It's good to know that everything was wrong, I mean, not in compliance with the rules, thanks